### DISCUSSION TITLE
What is the "unofficial and minimal Realtek UAD generic package"?

### DISCUSSION THREAD

**Discussion by kingmustard** [pal1000/Realtek-UAD-generic#90] [Q&A] [answered]
Part way through installation, the script asks: "Do you want to install unofficial and minimal Realtek UAD generic package?"

What is this?

**Comment by pal1000** [comment, marked as answer]
It's the full name of this software package.

**Reply by kingmustard** [reply]
> It's the full name of this software package.

Got you.

So basically, choosing No will stop the script/installation process?

**Reply by pal1000** [reply]
Yes and after that you can return to the your official OEM audio driver if you want.
